### ISSUE TITLE
Number of epochs for warm up training on source

### ISSUE THREAD

**Issue by BjoernMichele** [saltoricristiano/cosmix-uda#6]
Dear @saltoricristiano, 

for the warm-up training on the source I am a bit confused which paramteres to use. In the paper you mention 10 epochs, but in the config files in the repo it is mentioned 100 epochs. Also I would be interested on the batch size for the warm up training, is it 4 ? 

Thank you very much :) 

Best, 
Bjoern

**Comment by saltoricristiano** [comment]
Hi @BjoernMichele,

Yes, we warm up for 10 epochs and then use the pretrained models for adaptation. The provided models are pre-trained for 10 epochs.
During pre-training we use a batch size of 4.

I'm going to correct the num of epochs as I was stopping by hand the trainings after 10 epochs (you can try to train even more but I did not get any significant improvement neither on source nor on target).

Did I solve your doubts?
Cristiano

**Comment by BjoernMichele** [comment]
Dear @saltoricristiano , 

again thank you very much for your prompt reply. All doubts are solved :) 

Best, 
Bjoern
